root/defaults/peer.conf

```diff
@@ -1,5 +1,6 @@
 [Interface]
 Address = ${CLIENT_IP}
+PresharedKey = $(cat /config/${PEER_ID}/presharedkey-${PEER_ID})
```

Comment:
PresharedKey need to be moved down to [Peer] section.

Reply:
good eye, thanks